alchemist-multivesta-adapter/src/main/kotlin/it/unibo/alchemist/multivesta/adapter/AlchemistMultiVesta.kt

```diff
@@ -47,7 +50,6 @@ object AlchemistMultiVesta {
     private const val MULTIVESTA = "mv"
     private const val ALREADY_EXECUTED_SIMULATION = "aes"
     private const val HEADLESS = "hl"
-    private const val VARIABLES = "var"
     private const val BATCH = 'b'
     private const val FXUI = "fxui"
```

Comment:
This could be tropped, right?

Reply:
This is a mistake, I have reverted the changes and put the variables back in. Please re-review this file.